AElf.OS.Network.Grpc/GrpcUrl.cs

```diff
@@ -17,6 +17,11 @@ public static GrpcUrl Parse(string address)
             return url;
         }
         
+        public string ToIpPortFormat()
```

Comment:
You can use System.Uri or System.Net.IpAddress

no need to implement by your self

Reply:
@samfrb

Reply:
will be solved by #940